package.json

```diff
@@ -41,8 +41,8 @@
     "@graphql-codegen/typescript-operations": "^1.13.1",
     "@manifoldco/eslint-plugin-stencil": "^0.4.1",
     "@manifoldco/icons": "0.0.6",
-    "@manifoldco/manifold-init": "^0.3.0",
-    "@manifoldco/manifold-init-types": "^0.3.0",
+    "@manifoldco/manifold-init": "^0.4.0-beta.1",
+    "@manifoldco/manifold-init-types": "^0.4.0-beta.1",
```

Comment:
@sslotsky I added `@manifoldco/manifold-init: *` to peerDeps yesterday. WDYT?

Reply:
I think that's a good call! Anyone who bundles this from NPM would benefit from that peer dependency warning.